docs/algorithms/AlphaShape.asciidoc

```diff
@@ -7,14 +7,14 @@ data set is far superior to another this can improve the conflation quality and
 reduce amount of time reviewing.
 
 Hootenanny uses the
-https://insightcloud.digitalglobe.com/redmine/attachments/download/2858/2011-B-03-AlphaShapes.pdf[alpha
+link:$$https://github.com/ngageoint/hootenanny/wiki/files/2010-B-01-AlphaShapes.pdf$$[alpha
 shape] algorithm to define the shape of a finite set of points. In this case we
 treat the vertexes of the input shapefile as the set of points when calculating
 the alpha shape. 
 
 [[alphashape]]
 .Simple Alpha Shape
-image::AlphaShape.png[A simple example of an alpha shape,scalewidth="50%"]
+image::algorithms/images/AlphaShape.png[A simple example of an alpha shape,scalewidth="50%"]
```

Comment:
This is the biggest change.  Almost no images were showing because they were moved to the subdirectories.  These changes fix that.